css-typed-om/Overview.bs

```diff
@@ -784,6 +787,72 @@ The following are the arithmetic operations you can perform on dimensions:
         is set to |this|.
 </div>
 
+<div algorithm="CSSNumericValue.min()">
+    The <dfn for=CSSNumericValue method>min(...|values|)</dfn> method,
+    when called on a {{CSSNumericValue}} |this|,
+    must perform the following steps:
+
+    1. Replace each [=list/item=] of |values|
+        with the result of [=rectifying a numberish value=] for the [=list/item=].
+
+    2. Prepend |this| to |values|.
+
+    3. If there are at least two different [=strong types=]
+        among the [=list/items=] in |values|,
+        [=throw=] a {{TypeError}}.
```

Comment:
Throwing a TypeError is correct. You can't compare a length to an angle; doing so will be rejected as a syntax error in CSS, and throws a TypeError here in JS.